File: c/Angle Between Hands of a Clock.c
double angleClock(int hour, int minutes){
    double angle;
	    angle = (hour*30 + minutes*0.5) - (minutes*6);
	    if(angle<0){
	        angle = -1 * angle;
	    }
	    
	    if (360-angle < angle){
	        return (360-angle);
	    }

	    return(angle);

}

File: python/Angle Between Hands of a Clock.py
class Solution(object):
    def angleClock(self, hour, minutes):
        """
        :type hour: int
        :type minutes: int
        :rtype: float
        """
        angle = abs((hour*30 + minutes*0.5) - (minutes*6))

        return(min(360-angle,angle))
        
